### DISCUSSION TITLE
👀 Turbo & Contentlayer Issue

### DISCUSSION THREAD

**Discussion by Pyr33x** [vercel/community#5440] [Bug Report] [answered]
### Page to Investigate

https://nextjs.org/docs/architecture/turbopack

### Steps to Reproduce

Hey,
I'm currently using `next v14.0.4` and `contentlayer v0.3.4`, and cl doesn't support next 14 yet but it still work fine in development/production, since I add `--turbo` to the next dev prefix, it doesn't generate/update the content anymore, and that depends on **contentlayer** to diagnose the changes in **MDX**

**Expectations**
- Add **Turbo** Support for **Contentlayer**

**Comment by versecafe** [comment, marked as answer]
This is likely related to this issue https://github.com/vercel/turbo/issues/6365 where turborepo fails to invalidate the cache over mdx file changes, likely occurring in turbopack due to shared logic and code, or as Anthony points out some internal issues with internal webpack cache failing to invalidate.

## Likely Fix
Ensure that mdx files are seen as a build/dev dependency and rebuild is triggered on hash change.

**Reply by Pyr33x** [reply]
Got it!
